test/sample_default_notebooks.tsx

```diff
@@ -3,6 +3,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+export const emptyNotebook = {
```

Comment:
Is there a reason these constants shouldn't also be in `notebooks_constants`?

Reply:
updated. These came from legacy code tests in notebooks. Moved them to `notebooks_constants`.